cli/installer/util.go

```diff
@@ -63,10 +63,10 @@ func detectPkgManager() pkgManager {
 	}
 
 	switch true {
-	case commandExists("brew"):
-		lastDetectedPkgManager = homebrew
 	case commandExists("apt-get"):
 		lastDetectedPkgManager = apt
+	case commandExists("brew"):
```

Comment:
I assume the same problems wuold happen on any linux with brew, so maybe it's better to move it below all linux package managers?